### DISCUSSION TITLE
Service Bot - Integration

### DISCUSSION THREAD

**Discussion by roliveira3891** [salman0ansari/whatsapp-api-nodejs#516] [General]
Hi!

I need to integrate with some system for attendance.

For those of you who have more experience, do you know any libraries or bots that can be integrated? Or where would I start?

Thanks

**Comment by ailton-tim** [comment]
Hi @roliveira3891 you can use these apps: 
www.make.com,  www.zapier.com or www.pluga.com.br (Integrations software) or can use www.landbot.io (Chatbot).
